source/install/deploy-bitnami.rst

```diff
@@ -0,0 +1,15 @@
+.. _deploy-bitnami:
+
+Deploy Mattermost on Bitnami
+=============================
+
+Through our partnership with Bitnami you can automatically deploy Mattermost Team Edition to a demonstration server for up to 1 hour, up to 10 times per platform, including: 
+
+- `Amazon Web Services <https://about.mattermost.com/deploy-aws>`_
+- `Oracle Cloud Platform <https://about.mattermost.com/deploy-oracle>`_ 
+- `Microsoft Azure <https://about.mattermost.com/deploy-azure>`_
+- `Google Cloud Platform <https://about.mattermost.com/deploy-googlecloud>`_
+
+Alternatively, you may also deploy to `Debian 8 VMware or Oracle VirtualBox virtual machines <https://about.mattermost.com/deploy-debianvirtualmachine>`_.
+
+If you have an existing account, or choose to create a new account, you can run your auto-deployed Mattermost instance as long as you would like.
```

Comment:
This last statement implies that you can get Mattermost on AWS for an hour without having an AWS account?

How do you get an hour's worth of AWS without having an account? I tried, but you can't use AWS without an account. So it looks like the 1 hour free thing does not apply to AWS? I looked briefly at the others, but it didn't look like they allowed non-account holders access to their infrastructure either. Do you have instructions on how to get this deal? I couldn't find anything about this in the Bitnami docs either.